sys/linux/sys.txt

```diff
@@ -96,8 +96,10 @@ writev(fd fd, vec ptr[in, array[iovec_in]], vlen len[vec])
 pwritev(fd fd, vec ptr[in, array[iovec_in]], vlen len[vec], off_low int32, off_high int32)
 pwritev2(fd fd, vec ptr[in, array[iovec_in]], vlen len[vec], off_low int32, off_high int32, flags flags[rwf_flags])
 lseek(fd fd, offset fileoff, whence flags[seek_whence])
+copy_file_range(fd_in fd, off_in ptr[inout, fileoff[int64]], fd_out fd, off_out ptr[inout, fileoff[int64]], len intptr, flags flags[copy_file_range_flags])
```

Comment:
Having null `off_in` and/or `off_out` seems to be valid: 
- https://man7.org/linux/man-pages/man2/copy_file_range.2.html
- https://elixir.bootlin.com/linux/latest/source/fs/read_write.c#L1769

Thus, we might want to make them optional:



```suggestion
copy_file_range(fd_in fd, off_in ptr[inout, fileoff[int64], opt], fd_out fd, off_out ptr[inout, fileoff[int64], opt], len intptr, flags flags[copy_file_range_flags])
```

Reply:
Yes, NULL is a valid argument, I added the suggested change!